examples/index.json

```diff
@@ -1035,6 +1015,32 @@
             }
         ]
     },
+    {
+        "title": "Access Control",
+        "column": 3,
+        "category": "Working over the network",
```

Comment:
```suggestion
        "category": "Working via the network",
```

Reply:
That is the main category name. So won't apply this change.